.github/workflows/ci.yaml

```diff
@@ -40,6 +40,10 @@ jobs:
           - name: 'Unit Tests'
             task: 'ci-unit'
             python-version: '3.6'
+          # This job is slow so we only run in on a daily basis
```

Comment:
Github actions works much better if we use more workflows because you can only restart jobs at the Workflow level, not per job.

So, when we add the benchmark job, it should be in a separate Workflow file.

Reply:
ack. I will likely add it in the future to run once a week or something since the job is very slow on CI (~30 minutes).